experimental/packages/opentelemetry-instrumentation-http/package.json

```diff
@@ -9,7 +9,9 @@
     "prepublishOnly": "npm run compile",
     "compile": "tsc --build",
     "clean": "tsc --build --clean",
-    "test": "nyc mocha test/**/*.test.ts",
+    "test:cjs": "nyc mocha test/**/*.test.ts",
+    "test:esm": "nyc node --experimental-loader=@opentelemetry/instrumentation/hook.mjs ../../../node_modules/.bin/mocha 'test/**/*.test.mjs'",
+    "test": "npm run test:cjs && npm run test:esm",
```

Comment:
Reviewer note: This `test:esm` script follows the same pattern used for ESM tests in opentelemetry-instrumentation.